apps/studio/pages/project/[ref]/auth/mfa.tsx

```diff
@@ -0,0 +1,49 @@
+import { PermissionAction } from '@supabase/shared-types/out/constants'
+
+import { MfaAuthSettingsForm } from 'components/interfaces/Auth'
+import AuthLayout from 'components/layouts/AuthLayout/AuthLayout'
+import {
+  ScaffoldContainer,
+  ScaffoldDescription,
+  ScaffoldHeader,
+  ScaffoldTitle,
+} from 'components/layouts/Scaffold'
+import NoPermission from 'components/ui/NoPermission'
+import { GenericSkeletonLoader } from 'components/ui/ShimmeringLoader'
+import { useCheckPermissions, usePermissionsLoaded } from 'hooks/misc/useCheckPermissions'
+import type { NextPageWithLayout } from 'types'
+
+const AuthSettings: NextPageWithLayout = () => {
+  const isPermissionsLoaded = usePermissionsLoaded()
+  // TODO: check if these permissions cover third party auth as well
+  const canReadAuthSettings = useCheckPermissions(PermissionAction.READ, 'custom_config_gotrue')
+
+  return (
+    <>
+      <ScaffoldContainer>
+        <ScaffoldHeader>
+          <ScaffoldTitle>Multi-factor Authentication (MFA)</ScaffoldTitle>
```

Comment:
likewise - wondering if we should match the page title and side menu label here
![image](https://github.com/user-attachments/assets/9fa17926-55c9-499c-a1f9-f0edc2cec05d)
![image](https://github.com/user-attachments/assets/9760f3d6-eb0a-4585-8b17-a974f57050cd)

Reply:
Nit: Casing consistency for multi-factor
![image](https://github.com/user-attachments/assets/de6ca857-d1cc-4ee4-8d7f-65d00cfda4d2)
![image](https://github.com/user-attachments/assets/82cc077e-6387-44ef-ada5-655f06d4b66b)